webpack.config.js

```diff
@@ -1,6 +1,12 @@
 const path = require('path');
 const webpack = require('webpack');
 
+var version = require('./package.json').version;
```

Comment:
minor issue:
- can we use string literals for this?
- can we extract this to a `/scripts/banner.js` and use it from both webpack configs?